public/_templates/bulma/css/forum.css

```diff
@@ -109,3 +109,11 @@ a.neutral-link {
     hyphens: auto;
   }
 }
+
+/* More wide searchbar */
```

Comment:
Just a small wording suggestion:

```suggestion
/* Widen search bar to fill available space */
```

Reply:
Thanks for your suggestion! Fix coming soon...